apps/web/server/lib/router/getServerSideProps.ts

```diff
@@ -8,12 +9,26 @@ import { TRPCError } from "@trpc/server";
 export const getServerSideProps = wrapGetServerSidePropsWithSentry(async function getServerSideProps(
   context: GetServerSidePropsContext
 ) {
+  const isEmbed = hasEmbedPath(context.req.url || "");
+
+  if (process.env.NEXT_PUBLIC_IS_E2E && process.env.VERCEL_BOTID_ENABLED) {
```

Comment:
I think this should be `!process.env.NEXT_PUBLIC_IS_E2E_&& process.env.VERCEL_BOTID_ENABLED`